docs/_news/news07.md

```diff
@@ -20,14 +20,16 @@ Curious? [Learn more](remoteoverview.html)!
 
 #### New Features and Highlights for 0.7.0
 
+**Monitoring Updates**
+- You can now [inject metrics](https://github.com/codewind-resources/design-documentation/blob/master/codewindServer/metricsInjection.md) into your Node.js, Liberty, and Spring projects. 
+- Effortlessly [visualize data](https://github.com/eclipse/codewind/issues/977) like CPU usage, memory footprint, and web traffic on the included application monitoring dashboard at the click of a button. If you don't want it or need it, simply [disable it](https://github.com/eclipse/codewind/issues/1290). Codewind hosts the dashboard, so displaying the graphs won't eat into your project's resources.
+
 **OpenShift Do (odo)**
 - You can now easily import existing projects into Codewind and [use odo](https://github.com/eclipse/codewind/issues/1115) to build and deploy projects to your OpenShift or OKD cluster.
-- Introducing [support for Java template](https://github.com/eclipse/codewind/issues/450) and Java projects can build and deploy [once configured](//permalink to landing page).
+- Introducing [support for Java template](https://github.com/eclipse/codewind/issues/450) and Java projects can build and deploy [once configured](mdt-che-odo-support.html).
```

Comment:
- How about, "Codewind now supports Java templates, and Java projects can build and deploy after they are configured."

Reply:
Integrated your change, small change to suggestion "after they are configured" >> "after they have been configured"